pkgs/development/compilers/futhark/update

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p cabal2nix
+
+echo Updating futhark package.nix file..
+
+expr="(import ../../../.. {}).callPackage ./src.nix {}"
+src=$(nix-build --no-out-link -E "$expr")
+if cabal2nix "$src" --hpack > tmp.nix; then
+	mv tmp.nix package.nix
+else
+	rm tmp.nix
+	exit 1
+fi
+
+echo Updated package.nix file successfully
```

Comment:
I feel like if you have to use this script then it should be packaged properly and automatically run on whatever `src` you choose rather than just living in the directory.